packages/console/src/components/Search/index.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { FormEventHandler, KeyboardEventHandler, useState } from 'react';
+
+import SearchIcon from '@/icons/Search';
+
+import Button from '../Button';
+import TextInput from '../TextInput';
+import * as styles from './index.module.scss';
+
+type Props = {
+  defaultValue?: string;
+  onSearch?: (value: string) => void;
+};
+
+const Search = ({ defaultValue = '', onSearch }: Props) => {
+  const [inputValue, setInputValue] = useState<string>(defaultValue);
```

Comment:
If we use a state and set `defaultValue` as the initial state, then it won't change when the defaultValue changes from upperstream.
Is this by design? If not, then we should consider using a `useEffect` to update `inputValue` state